README.md

```diff
@@ -101,6 +101,37 @@ poetry install
 
 </details>
 
+<details>
+  <summary>Option 3️⃣ : Docker </summary>
+
+#### Nvidia docker dependency
+  To run the Docker image with GPU support, ensure that you have the NVIDIA Container Toolkit installed. Detailed installation instructions can be found here - https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html.
+
+#### Pull the image and run the container
+To pull our pre-built Docker image and begin using GPUDrive, execute the following command (you may need to prepend sudo, depending on your Docker setup):
+  ```bash
+  docker pull ghcr.io/emerge-lab/gpudrive:latest
+  ```
+
+After pulling the image, you can create and run a new container using the --gpus all flag (omit this flag if your system does not have a GPU):
```

Comment:
Has this been tested yet? I can test this later today. Additionally, shall we ask someone else that is working on another machine?

Reply:
I tested it on Emerge2, but yes definitely test it on a different machine.